packages/ui/tailwind-design-system.ts

```diff
@@ -108,6 +112,24 @@ export default {
           success: 'hsl(var(--canary-foreground-success))',
           accent: 'hsl(var(--canary-foreground-accent))'
         },
+        'cn-foreground': {
```

Comment:
New design system classes will have `cn-` as their prefix, followed by their equivalent name as in existing design system.

Example:
`foreground` -> `cn-foreground`